packages/webapp/src/routes/land/download-legal-agreement-file.js

```diff
@@ -2,20 +2,24 @@ import path from 'path'
 import { blobStorageConnector } from '@defra/bng-connectors-lib'
 import constants from '../../utils/constants.js'
 import { logger } from 'defra-logging-facade'
-
-const downloadLegalAgreementFile = async (request, h) => {
-  const blobName = request.yar.get(constants.redisKeys.LEGAL_AGREEMENT_LOCATION)
-  const config = {
-    blobName,
-    containerName: 'trusted'
+const handlers = {
+  get: async (request, h) => {
+    const { id } = request.query
```

Comment:
Should Get route have some query string validation for security?

Reply:
new ticket